client/controllers/userEvent.coffee

```diff
@@ -14,6 +14,26 @@ Template.userEvent.onRendered ->
 Template.userEvent.helpers
   isEditing: ->
     return Template.instance().editState.get()
+  incidentLocations: ->
+    locations = {}
+    counts = Template.instance().data.counts
+    # Loop 1: Incident Reports ("counts")
+    for count in counts
+      # Loop 2: Locations within each "count" record
+      for loc in count.locations
```

Comment:
Ok, I'll introduce the fixes. But just as a side note, this helper won't be staying in the userEvent controller for very long. The map is going to be moved to the locationList template in #43.